HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -557,14 +557,48 @@ def self.run_validator(hpxml_doc)
       '/HPXML/Building/BuildingDetails/Systems/MechanicalVentilation/VentilationFans/VentilationFan[UsedForWholeBuildingVentilation="true"]' => {
         'SystemIdentifier' => one, # Required by HPXML schema
         'FanType[text()="energy recovery ventilator" or text()="heat recovery ventilator" or text()="exhaust only" or text()="supply only" or text()="balanced" or text()="central fan integrated supply"]' => one, # See [MechVentType=HRV] or [MechVentType=ERV] or [MechVentType=CFIS]
-        'TestedFlowRate | RatedFlowRate' => one_or_more,
         'HoursInOperation' => one,
+        'IsSharedSystem' => one, # See [SharedSystem]
+      },
+
+      ## [SharedSystem=true]
+      '/HPXML/Building/BuildingDetails/Systems/MechanicalVentilation/VentilationFans/VentilationFan[UsedForWholeBuildingVentilation="true" and IsSharedSystem="true"]' => {
+        'TestedFlowRate[@scope="single unit"] | RatedFlowRate[@scope="single unit"]' => one_or_more, # Use scope attribute to describe at which level data is provided
+        'TestedFlowRate[@scope="multiple units"] | RatedFlowRate[@scope="multiple units"]' => one_or_more, # Use scope attribute to describe at which level data is provided
+        'FanPower[@scope="multiple units"] | FanPower[@scope="single unit"]' => one_or_more, # Use scope attribute to describe at which level data is provided
+        'FanPower[not(@scope)]' => zero, # Use scope attribute to describe at which level data is provided
+        'FractionOutdoorAir | FractionRecirculation' => one_or_more,
```

Comment:
I suggest just asking for one of these, not both, since they need to sum to 1 (and may be off due to rounding). No strong preference but I guess I'd choose FractionRecirculation.